lib/active_merchant/billing/gateways/netbanx.rb

```diff
@@ -132,6 +132,11 @@ def add_credit_card(post, credit_card, options = {})
         if options[:billing_address]
           post[:card][:billingAddress]  = map_address(options[:billing_address])
         end
+
+        if options[:authentication]
```

Comment:
```suggestion
        if options[:three_d_secure]
```

You can refer to the 3DS fields from this doc:
https://github.com/activemerchant/active_merchant/wiki/Standardized-3DS-Fields

Reply:
Updated